components/runtime.rst

```diff
@@ -462,10 +458,23 @@ always using this ``ReactPHPRunner``::
 
 The end user will now be able to create front controller like::
 
-    require_once dirname(__DIR__).'/vendor/autoload_runtime.php';
+    use App\Runtime\ReactPhpRuntime;
+    use Psr\Http\Server\RequestHandlerInterface;
 
-    return function (array $context): SomeCustomPsr15Application {
-        return new SomeCustomPsr15Application();
+    $_SERVER['APP_RUNTIME'] = ReactPhpRuntime::class;
+
+    require_once dirname(__DIR__) . '/vendor/autoload_runtime.php';
+
+    return static function (): RequestHandlerInterface {
+        return new class implements RequestHandlerInterface {
+            public function handle(\Psr\Http\Message\ServerRequestInterface $request): \Psr\Http\Message\ResponseInterface
```

Comment:
Can we please use use statements? Thanks

Reply:
Thanks a lot, I remove this part